website/datataking.html

```diff
@@ -73,31 +74,57 @@ <h2 id="centerheader" style= "border:solid;border-color:black;" font-family:"cou
+	  <br>
+		<p><span class="boldfont"> Finding RTL-SDR Block</span></p>
+	<img src="https://astrochart.github.io/website/tutorials/firstdata1.png" class="left" alt="RTL_SDR Block" style="height:200px;border-radius:5px;">
+	<br>
+		<p><span class="boldfont">Finding Sink Block</span></p>
+	<br>
  <img src="https://astrochart.github.io/website/tutorials/firstdata2.png" class="left" alt="GUI Sink Block" style="height:250px;border-radius:5px;">
- <br><span class="boldfont">Connecting the blocks to eachother so that they can interact with eachother.</span></br>
-  <img src="https://astrochart.github.io/website/tutorials/firstdata3.png" class="left" alt="Connecting the Blocks" style="height:430px;border-radius:5px;">
-
-    <br>  I also modified my default Variable box to a Value: 1M, but this is only a personal preference. </br>
-<br>2.Connect radio antenna source to the RTL-SDR input so that we can see real radio data in our program. Make sure to elongate the antennas fully so that you can receive a strong radio signal. </br>
-<br><span class="boldfont">How the Hardware Setup Should Look. </span></br>
-<img src="https://astrochart.github.io/website/tutorials/firstdata4.JPG" class="left" alt="Antenna set Up" style="height:450px;border-radius:5px;"></br>
-<br>3. Click the “Generate the flow graph” button, to transform our flow chart into python code. Next click the “Execute the flow graph button” to create a real time data plot of the incoming radio signals.</br>
-<br><span class="boldfont">Where to Find the "Execute" "Generate" Flow Graph Buttons.</span></br>
-  <img src="https://astrochart.github.io/website/tutorials/firstdata5.png" class="left" alt="Genterate the Flow Graph" style="height:350px;border-radius:5px;"></br>
-  <br>4. A new window will pop up that displays incoming radio data. You can click on the tabs at the top of the screen to view the data in “Frequency”, “Waterfall”, “Time Domain”, or “Constellation” display.</br>
-  <br><span class="boldfont">This is the data that we collected.</span></br>
-<img src="https://astrochart.github.io/website/tutorials/firstdata6.png" class="left" alt="First Data Set" style="height:400px;border-radius:5px;"></br>
-Congratulations, you have collected your first radio data set!
+ <br>
+		<p><span class="boldfont">Connecting the blocks so that they can interact with each other.</span></p>
+	<img src="images_for_website/ScreenshotsFromPi/addedHeadBlock.png" class="left" alt="Connecting the Blocks" style="height:430px;border-radius:5px;">
+		<h5>2. Set the correct variable numbers to change how frequently we will sample data.</h5>
+	<p>We want to change the variable value to 1,000,000.
+	  <li>Double click on the variable block and type in 1000000 and select okay.</li></p>
+		<br>
+		<p>We also want to change the center frequency to 100,000,000.
+		<li>Double click the QT GUI Sink and change the <i>Center Frequency (Mz)</i> to 100000000 and select okay.</li></p>
```

Comment:
Double check, but I think it should be Hz, not Mz.
```suggestion
		<li>Double click the QT GUI Sink and change the <i>Center Frequency (Hz)</i> to 100000000 and select okay.</li></p>
```